### ISSUE TITLE
Collapsible alerts the severity or type of alert is not being convayed to  screen reader users : 

### ISSUE THREAD

**Issue by dfarough** [wet-boew/wet-boew#7662]
@wet-boew/access-working-group

given that a css image and different background colours are used to convey the importance of the alert messages, I feel that we need to find a better way to present this information to users of access technology.  could we not use an image with the appropriate alt text instead of the CSS image which is invisible to Access  technologies?

As I see it, there is no way for a screen reader user to know what type a message they are hearing unless the text of the message makes it obvious.

As it stands now I thing that these alerts fail WCAG 2.0 using the following criteria:

1.1.1 Non-text Content: All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, 
1.4.1 Use of Color: Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element. (Level A)

**Comment by kpduncan** [comment]
These is my personal approach to working around this issue:

For normal alerts:

```html
<span id="info" class="wb-sl">This is an information alert providing extra details about usability on the page.</span>
<section class="alert alert-info" aria-describedby="info" >
<h3></h3>
<p></p>
</section>
```
If you want the alert to be read on page load add `role="alert"`.

For collapsible alerts I use:

```html
<section>
<span id="warning" class="wb-sl">This is an warning alert! Providing important details about usability on the page.</span>
	<details class="alert alert-warning" id="alert-warning" open="open" aria-describedby="warning" >
		<summary class="h3"><h3></h3></summary>
		<p></p>
	</details>
</section>
```
